app-backend/http4s-util/src/main/scala/com/rasterfoundry/http4s/Config.scala

```diff
@@ -1,13 +1,35 @@
 package com.rasterfoundry.http4s
 
+import com.amazonaws.xray.plugins.{EC2Plugin, ECSPlugin}
 import com.typesafe.config.ConfigFactory
+import scala.collection.JavaConverters._
+import com.rasterfoundry.common.{Config => CommonConfig}
 
 object Config {
   private val config = ConfigFactory.load()
 
+  val environment = CommonConfig.awsbatch.environment
+
   object cache {
     private val cacheConfig = config.getConfig("cache")
     val authenticationCacheEnable =
       cacheConfig.getBoolean("authenticationCacheEnable")
   }
+
+  object ecs {
```

Comment:
I tested both of these locally and they don't error (just empty) so I think it's OK to add these here. Should give us some additional metadata once this deployed in an AWS environment.